certified-connectors/Encodian/apiDefinition.swagger.json

```diff
@@ -11143,26 +12288,21 @@
     "DtoPopulateWordDocument": {
       "required": [
         "documentData",
-        "fileContent"
+        "fileContent",
+        "populationOptions"
```

Comment:
@Encodian The input field has been changed from optional to required. This might break existing flows. can you please confirm that this will not affect existing users.

Reply:
Setting the 'populationOptions' property to required is needed because this is now an internal property. It will not break existing flows as this is an enum and an empty value would have generated a 400 error anyway... considering there is no change in behavior by setting this to required.